### PR TITLE
Added in french translation of Paris bootcamp page.

### PR DESCRIPTION
**Pull request by pipitone**
Some parts came from @ccossou's PR #239, and others I translated myself.  Should probably have someone fluent in French double check my work.

### CODE REVIEW

**Comment by adamobeng** [comment]
I am fluent in French, but my technical French is pretty poor. I'll have a go if you want.

How do I submit a diff to a pull request?

**Comment by pipitone** [comment]
@adamobeng... I don't think you can (may be wrong on this one though).  You can always submit your own pull request and reference this one.  Or submit a pull request to my paris-bootcamp-fr branch, and I can push the changes here.

File: bootcamps/2013-04-paris.html [hunk 1]
```diff
@@ -50,4 +50,56 @@
...
+  Les participants seront encourag&eacute;s &agrave; s'entraider et &agrave;
+  appliquer ce qu'ils ont appris &agrave; leurs propres probl&egrave;mes de
+  recherche pendant et en dehors des sessions. Il y aura des sessions de suivi
+  en ligne pendant 6 &agrave; 8 semaines afin de compl&eacute;ter et renforcer
+  les bases enseign&eacute;es lors de la session d'entra&icirc;nement.
+</p>
+<table>
+  <tr>
+    <td>1er jour</td>
+    <td>du matin</td>
```
Comment (agramfort) [inline bootcamps/2013-04-paris.html]: "le matin" sounds more french

File: bootcamps/2013-04-paris.html [hunk 2]
```diff
@@ -50,4 +50,56 @@
...
+    <td>...</td>
+    <td>Introduction &aacute; Python</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>de l'apr&eacute;s-midi</td>
```
Comment (agramfort) [inline bootcamps/2013-04-paris.html]: L'après-midi

File: bootcamps/2013-04-paris.html [hunk 3]
```diff
@@ -50,4 +50,56 @@
...
+    <td>...</td>
+    <td>NumPy pour le calcul scientifique</td>
+  </tr>
+  <tr>
+    <td>2e jour</td>
+    <td>du matin</td>
```
Comment (agramfort) [inline bootcamps/2013-04-paris.html]: le matin

File: bootcamps/2013-04-paris.html [hunk 4]
```diff
@@ -50,4 +50,56 @@
...
+    <td>...</td>
+    <td>Gestion de versions avec Git</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>de l'apr&eacute;s-midi</td>
```
Comment (agramfort) [inline bootcamps/2013-04-paris.html]: l'après-midi

File: bootcamps/2013-04-paris.html [hunk 5]
```diff
@@ -50,4 +50,56 @@
...
+    <td>...</td>
+    <td>Test de logiciels scientifiques</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>..</td>
+    <td></td>
+    <td>La structuration, la documentation et l'emballage de votre projet</td>
```
Comment (agramfort) [inline bootcamps/2013-04-paris.html]: I don't like the word emballage in this context. I would say la "distribution". Meaning is a bit different but that's the same objective.

**Comment by agramfort** [comment]
Thanks

**Pull request merged**